src/commands/ban_commands.py

```diff
@@ -21,10 +21,19 @@ async def ban(
         reason: str,
     ):
         """Ban the specified user."""
-        async with create_response_context(interaction) as response_message:
-            async with create_logging_embed(
-                interaction, user=user, reason=reason
-            ) as logging_embed:
-                await ban_user(logging_embed, user, reason)
+        # result is a tuple of state and descriptio
+        result = await ban_user(user, reason)
```

Comment:
accessing result directly is hard to read in dense code like this, so let's initialize some vars before we go into the if statements